### DISCUSSION TITLE
Oracle cloud issue [Apr 05]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1665] [Discord Threads]
> 🙋‍♂️ **PoisonFlame**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Surely memory is not the issue. NocoDB would be fine with as little as 128-256MB  
#  
> 💁 **o1lab**  
#### https://github.com/nodejs/docker-node/issues/1559  
#  
> 🙋‍♂️ **PoisonFlame**  
#### Ah, yes, I came across that a little while after I made the post. I tried the possible solution as well as the stuff that person mentioned in the "Additional Information" and it does seem like its a node 12 alpine issue.   
####    
#### I'm now going through the git route (made a container using `node:14-alpine` as base that pulls and runs based on instructions here: https://github.com/nocodb/nocodb-seed)   
####    
#### I updated the nocodb version in the `package.json` and I was able to get the service running but I'm now having issues with setting it up with postgres. It seems like it's not communicating with postgres at all as I see no new tables created or any connection attempts in my postgres container. With that, I was able to create a user but I'm not sure where the credentials are being stored.   
####    
#### The way I'm connecting is by setting the `NC_DB` variable in my `.env` file which is in the root of the git project directory. My `NC_DB` is as follows: `NC_DB=pg://db:5432?u=user&p=pass&d=nocodb`   
####    
#### Also, somewhat separate from the above, but with the git installation route, which directories should I be backing up to save my data in case I need to reinstall or update the current install?   
####    
#### Thanks again.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/959905186162290778)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
